### PR TITLE
Fix branches presentation on PR's list

### PR DESCRIPTION
**Pull request by IT-AlexKor**
This is a hotfix for UI presentation of branches on PR's list (implemented in #19747)

Current representation (before fix)
before_fix

New representation
after_fix

**Please include/backport (if possible) to v1.17.2 for example**

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Comment by silverwind** [comment]
Truncation is good, but I don't agree it making it bold again, I think it look better without bold.

File: templates/shared/issuelist.tmpl [hunk 1]
```diff
@@ -66,17 +66,17 @@
 					{{if .IsPull}}
 						<div class="branches df ac">
 							<div class="branch">
-								<a href="{{.PullRequest.BaseRepo.HTMLURL}}/src/branch/{{PathEscapeSegments .PullRequest.BaseBranch}}">
+								<a class="bold dif" href="{{.PullRequest.BaseRepo.HTMLURL}}/src/branch/{{PathEscapeSegments .PullRequest.BaseBranch}}">
 									{{/* inline to remove the spaces between spans */}}
-									{{if ne .RepoID .PullRequest.BaseRepoID}}<span class="truncated-name">{{.PullRequest.BaseRepo.OwnerName}}</span>:{{end}}<span class="truncated-name">{{.PullRequest.BaseBranch}}</span>
+									{{if ne .RepoID .PullRequest.BaseRepoID}}<span class="truncated-name dib vb">{{.PullRequest.BaseRepo.OwnerName}}</span>:{{end}}<span class="truncated-name dib vb">{{.PullRequest.BaseBranch}}</span>
```
Comment (silverwind) [inline templates/shared/issuelist.tmpl]: `vb` does not appear to be a [valid helper class](https://github.com/go-gitea/gitea/blob/main/web_src/less/helpers.less).

**Review by silverwind — CHANGES_REQUESTED** [review]
I'm not sure how this truncation is supposed to work, maybe mobile layout only? I think generally we want to set `max-width` and use `.ellipsis` to limit to a certain length, but unless there is a strong reason to truncate the branch name, I wouldn't do it.

Branch name holds important information and if we can get it to wrap, it would be better than just cutting it off like that.

File: templates/shared/issuelist.tmpl (hunk 1 above)
Comment (IT-AlexKor) [reply templates/shared/issuelist.tmpl]: deleted

**Comment by IT-AlexKor** [comment]
@silverwind I will try to explain my decisions.

Why bolded branch names?
1) Easier to see - without bold it's look like "grey on grey" with a small color difference
2) Faster to find in a list - one of the scenarios in our teams is to review only PR's that are compared with the master branch only. So when the source/target branches are bolded it's easier to find requested PR's in list by eyes. Without bold whole list looks like a single huige block of text which hard to analyze fastly

Why truncated names?
1) Each PR in list already have many possible sub-items (num of approvers, num of change request, author, creation date and etc) and when they stick together the line is getting too long and line breaks. 
2) Before #19747 there are no information about source/target branches was present in list, so my decision didn't make it worse than before. Usually there are no truncating becuase branch name is already short.

About "max-width and use .ellipsis": if you know how to make it better please suggest your solution. I have a better experience in backend than in frontend.

**Comment by silverwind** [comment]
1. I don't agree with bold, sorry. We just removed the bold in https://github.com/go-gitea/gitea/pull/20726 and I think bold should be used very sparingly in UI, this is not a place for it.

2. `.truncated-name` is truncating to 10em, which makes even moderate branch names tuncated. I think it's truncating too soon to be any useful:
image

**Comment by IT-AlexKor** [comment]
I didn't know about #20726 and thought that original UI from #19747 was broken accidentally by somebody else.

Sorry for the inconvenience and thanks for your time and detailed explanation. I'm closing current PR (nothing to fix)

**Pull request closed without merging**